readme.md

```diff
@@ -99,10 +99,14 @@ let d = 0.0;
 The following are vaild negative Numbers:
```

Comment:
There’s a typo on this line that predates this pull, however it could probably be fixed in this pull.

Reply:
What is the typo?

Reply:
vaild vs valid

Reply:
Fixed